x-pack/plugins/gis/public/shared/layers/styles/vector_style.js

```diff
@@ -319,8 +303,8 @@ export class VectorStyle {
       mbMap.setPaintProperty(lineLayerId, 'line-opacity', 0);
     }
 
-    if (this._descriptor.properties.lineWidth && this._descriptor.properties.lineWidth.options) {
-      const lineWidth = this._getMbSize('lineWidth');
+    if (this._descriptor.properties.lineWidth) {
```

Comment:
any chance you could add a fix for https://github.com/elastic/kibana/issues/28866 here? that's a bad check, and bombs on 0-values.

Reply:
nvm, see https://github.com/elastic/kibana/issues/28866